sklearn/linear_model/_base.py

```diff
@@ -501,6 +505,26 @@ def fit(self, X, y, sample_weight=None):
         self : returns an instance of self.
         """
 
+        if self.normalize != "deprecate":
+            if not self.normalize:
+                warnings.warn(
+                    "'normalize' was deprecated in version 0.24 and will be"
+                    " removed in 0.26.", FutureWarning
+                )
+            else:
+                warnings.warn(
+                    "'normalize' was deprecated in version 0.24 and will be"
+                    " removed in 0.26. If you wish to keep an equivalent"
+                    " behaviour, use  Pipeline with a StandardScaler in a"
+                    " preprocessing stage:"
+                    "  model = make_pipeline( \n"
+                    "    StandardScaler(), \n"
+                    "    {type(self).__name__}())", FutureWarning
+                )
+            _normalize = self.normalize
+        else:
+            _normalize = False
```

Comment:
I would already put this in a private function that you can just call then for all the subsequent estimators.

`_normalize = _deprecate_normalize(self.normalize)`